src/components/StationListItem.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import StationModel from '../models/StationModels';
+
+interface StationListItemProps {
+    value: StationModel;
+    onClickSetValue: (value: StationModel) => void;
+}
+
+const StationListItem: React.FC<StationListItemProps> = ({ value, onClickSetValue }) => {
```

Comment:
I'd give `value` a more descriptive name so it's immediately obvious what `value` we are talking about